web-registry/src/pages/Dashboard/MyEcocredits/hooks/useFetchEcocredits.ts

```diff
@@ -27,9 +28,15 @@ interface Response {
 
 interface Props {
   address?: string;
+  creditClassId?: string;
```

Comment:
Updated the hook to be used with a `creditClassId`filter and without pagination.